### PR TITLE
feat: v4 Form component adoption

### PR DESCRIPTION
**Pull request by burakukula**

### CODE REVIEW

File: packages/markdown/src/dialogs/EmdebExternalContentDialog.tsx [hunk 1]
```diff
@@ -74,112 +67,110 @@ export const EmbedExternalContentModal = ({ onClose }: EmbedExternalContentModal
     }
   }, [mainInputRef]);
 
-  return (
```
Comment (gui-santos) [inline packages/markdown/src/dialogs/EmdebExternalContentDialog.tsx:77]: It seems that you are only removing the `()` here
Why?

File: packages/markdown/src/dialogs/InsertLinkModalDialog.tsx [hunk 2]
```diff
@@ -29,77 +29,75 @@ export const InsertLinkModal = ({ selectedText, onClose }: InsertLinkModalProps)
     }
   }, [mainInputRef]);
 
-  return (
```
Comment (gui-santos) [inline packages/markdown/src/dialogs/InsertLinkModalDialog.tsx:32]: same here

File: packages/markdown/src/dialogs/InsertTableModalDialog.tsx [hunk 3]
```diff
@@ -25,70 +25,68 @@ export const InsertTableModal = ({ onClose }: InsertTableModalProps) => {
     }
   }, [mainInputRef]);
 
-  return (
```
Comment (gui-santos) [inline packages/markdown/src/dialogs/InsertTableModalDialog.tsx:28]: ... and here

**Review by gui-santos — CHANGES_REQUESTED** [review]
Looks good, but I didn't understand why the `()` in some components was removed

File: packages/markdown/src/dialogs/EmdebExternalContentDialog.tsx (hunk 1 above)
Comment (burakukula) [reply packages/markdown/src/dialogs/EmdebExternalContentDialog.tsx:77]: hmm, you are right. I have run the codemod, didn't remove it manually. Something is wrong with the codemod? hmmmm, gonna check it. thanks!

Comment (massao) [reply packages/markdown/src/dialogs/EmdebExternalContentDialog.tsx:77]: try running the `prettier`, the default of the jscodeshift is not wrap single components in `()` in this case the single element being the `Fragment`.
But we have configured in prettier that it needs to, so prettier should fix it

**Review by massao — APPROVED** [review]

**Review by gui-santos — APPROVED** [review]

**Pull request merged**